pkg/acsengine/const.go

```diff
@@ -50,6 +50,8 @@ const (
 	NetworkPolicyCalico = "calico"
 	// NetworkPolicyCilium is the string expression for cilium network policy config option
 	NetworkPolicyCilium = "cilium"
+	// NetworkPolicyAzure is the string expression for cilium network policy config option
+	NetworkPolicyAzure = "azure"
 	// NetworkPluginAzure is the string expression for Azure CNI network policy
```

Comment:
This should probably be `network plugin`  instead of `network policy` since you are adding one for network policy. Or maybe we could reuse the same constant for both? Your call.

Reply:
@jackfrancis separated NetworkPolicy & NetworkPlugin. So basically NetworkPlugin means CNI and NetworkPolicy means network policy plugin. These are two different things.

Reply:
Yes I meant the comment should reflect that, right it's `// NetworkPluginAzure is the string expression for Azure CNI network policy` which is confusing